org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/preferences/DebugPreferencesMessages.properties

```diff
@@ -29,6 +29,7 @@ ConsolePreferencePage_12=Displayed &tab width:
 ConsolePreferencePage_13=Tab width must be between 1 and 100 inclusive.
 ConsolePreferencePage_11=Back&ground color:
 ConsolePreferencePage_Interpret_control_characters=Interpret ASCII &control characters
+ConsolePreferencePage_Force_system_encoding=Force system encoding by default (can be changed in launch configuration)
```

Comment:
Should we say it as "Force" or "Add" ?

Reply:
I prefer "force ... encoding" or "use ... encoding", "add" sounds strange, since we don't add encoding to console.
May be "use" is more neutral.

Reply:
"Use" sounds good.

Reply:
I see the words used in other apps as Automatic, Default and then the list of other available encodings.

Reply:
So I will change to "Use system encoding by default (can be changed in launch configuration)"